container/root/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,125 @@
+#############################################################
+# Replace nginx configuration directly in this file,
+# removing the need to perform error-prone replacements
+# at build time.
+#
+# For run-time replacements, ie, consuming environment vars,
+# add to the run.d/nginx script
+#############################################################
+
+user nobody;
+worker_processes auto;
+
+pid /tmp/nginx.pid;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    log_format main '$remote_addr - $remote_user [$time_local] "$request" '
+                    '$status $body_bytes_sent "$http_referer" '
+                    '"$http_user_agent" "$http_x_forwarded_for" NGINX_SERVER';
+
+    error_log /dev/stdout info;
+    access_log /dev/stdout main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    #keepalive_timeout  0;
+    keepalive_timeout  65;
+
+    #gzip  on;
+
+    client_body_temp_path /tmp/client_body;
```

Comment:
setup for non-root nginx master process. still a few more tweaks to make before it works correctly